src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/AesImplementation.Android.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Security.Cryptography
+{
+    internal sealed partial class AesImplementation
```

Comment:
Kevin: In theory this could be a partial to save the duplication, and AES-KW could be compiled just for OpenSSL.

Nivek: Yes but we should never have had Android try to pretend to be OpenSSL! We should split the PALs, not even further intertwine them!

Kevin: Hm. Yeah that is something I have thought from time to time. You make a pretty good point.

Nivek: I know. And it will allow these things to evolve independently, so while the source is similar now, it lays a good foundation for them going the direction each of them need to go.

Kevin: Okay, I agree. This is good as-is.